src/go/rdctl/pkg/snapshot/manager_test.go

```diff
@@ -31,20 +31,53 @@ func TestManager(t *testing.T) {
 		paths, _ := populateFiles(t, true)
 		manager := newTestManager(paths)
 		invalidNames := []string{
-			"test!",
-			`"test"`,
-			`'test'`,
-			"testtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest" +
-				"testtesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttesttest",
-		}
-		for _, c := range strings.Split("!$^&*()[]{};:?/'` \\\"", "") {
-			invalidNames = append(invalidNames, fmt.Sprintf("invalid%sname", c)) // spellcheck-ignore-line
+			"", // empty string not allowed
```

Comment:
Maybe include strings with other whitespace characters, at least try tabs and newlines.